src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +46,36 @@
      * Андрей - 600
      * София - 900</p>
      */
+    static DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
it isn't constant https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
add access modifier